### ISSUE TITLE
YouTube time of views doesn't grow during the script work

### ISSUE THREAD

**Issue by ice1x** [Ethical-H4CK3R/YouTubeViews#13]
Used cross platform script: python ./youtube.py 300 ../youtubeLinks.txt 77 10000

After a hours of usage i expected grow in the YouTube analytics related to time of views - but it still not increased.

I mean Youtube analytics -> Realtime data

**Comment by gogo43** [comment]
Yes me too

**Comment by LeandroHenriqueSS** [comment]
Same here

**Comment by gersanchez** [comment]
ahajajaja I think this does not work!

**Comment by Ethical-H4CK3R** [comment]
Yeah, they patched it. This program won’t work anymore, I will fix to find another way

**Comment by ice1x** [comment]
what about selenium webdriver?

**Comment by thepyguy** [comment]
What you need are dedicated proxies and Selenium. you can run selenium server or selenium grid with 10 or 20 or 50 or 100 session support. You will want to assign the same fingerprint to each IP. Same user agent, same browser, same IP. Then you should be able to provide views with each of these concurrent sessions on a timer. It takes quite a bit of resources to run 50 concurrent selenium instances and buy 50 ips. So is it worth?
